museum/sampleRecords/subjectList.destination.json

```diff
@@ -0,0 +1,169 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "occurrence_id":"1",
+      "id":"1",
+      "idno":"Anderson, ND",
+      "display_label":"Anderson, ND",
+      "ca_occurrences.type_id":"Person",
+      "ca_occurrences.idno":"Anderson, ND",
+      "ca_occurrences.preferred_labels":"Anderson, ND",
+      "ca_occurrences.LegacyID":"36",
+      "ca_occurrences.SubjectDates":"1910-1980",
+      "ca_occurrences.LastEditBy":"JILLM",
+      "ca_occurrences.LastEditDate":"1 January 2001",
+      "ca_objects.idno":"A1986.52",
+      "ca_objects.type_id":"Artwork",
```

Comment:
The `object_types` list in the profile has these values standardised against the list item (https://github.com/rwahs/installation-profile/blob/master/rwahs.xml#L50-L57). 

The importer does a case insensitive match on `idno`, and then on presentation it pulls out the label associated with the term.